.github/workflows/ci.yml

```
@@ -0,0 +1,21 @@
+name: Alfresco ROOT Web Application CI
+
+on:
+  pull_request:
+    branches:
+      - master
+      - support/**
+  push:
+    branches:
+      - master
+      - support/**
```

```suggestion
    branches:
      - master
  push:
    branches:
      - master
```


there are no support/ branches on this repo?